packages/rulesets/src/spectral/functions/get-collection-only-has-value-nextlink.ts

```diff
@@ -1,9 +1,6 @@
 // Verifies that collection get schemas have only the `value` and `nextLink` properties in their models.
 
 export const getCollectionOnlyHasValueAndNextLink = (properties: any, _opts: any, ctx: any) => {
-  if (!properties || typeof properties !== "object") {
```

Comment:
Why did this need to be removed?

Reply:
These lines never are never executed, we are sending properties object to this function.

Reply:
I think most of our functions have these checks at the beginning so that in case somehow a bad parameter gets passed in we won't end up with a null reference or some type error. I think we should keep it in here

Reply:
But when I leave the condition, as that is dead code unable to cover that in tests and build is failing due to test coverage percentage.

Reply:
lets add this back now and check if there is enough coverage?